microshift_install/microshift-install-rhel-for-edge.adoc

```diff
@@ -1,8 +1,23 @@
 :_content-type: ASSEMBLY
 [id="microshift-install-rhel-for-edge"]
-= Installing MicroShift in a RHEL for Edge Image
+= Embedding MicroShift into a RHEL for Edge Image
```

Comment:
@DanielFroehlich what's the right product term for RHEL for Edge? I thought we wanted to change that terminology?